layouts/_default/baseof.html

```diff
@@ -1,32 +1,40 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>{{ block "title" . }}{{ .Site.Title }}{{ end }}</title>
-    {{if .Site.Params.themeStyle}}
-      {{ partial (printf "%s/head.html" .Site.Params.themeStyle) . }}
-    {{else}}
-      {{ partial "flex/head.html" . }}
-    {{end}}
-  </head>
-  <body data-url="{{ .RelPermalink }}">
-    {{if .Site.Params.themeStyle}}
-      {{ partial (printf "%s/body-beforecontent.html" .Site.Params.themeStyle) . }}
-    {{else}}
-      {{ partial "flex/body-beforecontent.html" . }}
-    {{end}}
-    
+
+<head>
+  <title>{{ block "title" . }}{{ .Site.Title }}{{ end }}</title>
+  {{if .Site.Params.themeStyle}}
+  {{ partial (printf "%s/head.html" .Site.Params.themeStyle) . }}
+  {{else}}
+  {{ partial "flex/head.html" . }}
+  {{end}}
+</head>
+
+<body data-url="{{ .RelPermalink }}">
+  {{if .Site.Params.themeStyle}}
+  {{ partial (printf "%s/body-beforecontent.html" .Site.Params.themeStyle) . }}
+  {{else}}
+  {{ partial "flex/body-beforecontent.html" . }}
+  {{end}}
+
+  <article>
```

Comment:
previously, the `article` was part of the `body-beforecontent` and the `/article` was part of the `body-aftercontent`, which was terrifying and confusing.